docs/source/api_reference/regression.rst

```diff
@@ -30,3 +47,31 @@ Interval-based
     :template: class.rst
 
     TimeSeriesForestRegressor
+
+Kernel-based
+------------
+
+.. currentmodule:: sktime.regression.kernel_based
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    RocketRegressor
+
+Base
+----
+.. currentmodule:: sktime.regression
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    BaseRegressor
+
+.. currentmodule:: sktime.regression.deep_learning
+
+.. autosummary::
+    :toctree: auto_generated/
+    :template: class.rst
+
+    BaseDeepRegressor
```

Comment:
Both of these seem to be missing also. Maybe the whitespace is necessary?

Reply:
should be fixed